### DISCUSSION TITLE
提示账号异常

### DISCUSSION THREAD

**Discussion by wlwxj** [mikumifa/biliTickerBuy#439] [提问专区] [answered]
请求体: {'code': 100041, 'errno': 100041, 'errtag': 0, 'msg': '您的账号存在异常，暂时无法购票', 'data': {}}

**Comment by wlwxj** [comment]
![image](https://github.com/user-attachments/assets/a0867ae1-023a-4d05-a108-511dbcf377d8)

**Comment by mikumifa** [comment, marked as answer]
碰封了，不用抢一些不存在的票，不要提前抢，会被封

**Reply by nighttosakura** [reply]
我试了一下我两个账号抢已经开票的也出现了这种情况）

**Reply by mikumifa** [reply]
给个票id

**Reply by nighttosakura** [reply]
91975

**Reply by mikumifa** [reply]
你以前非法抢过票吗，会不会是以前被封禁的

**Reply by nighttosakura** [reply]
> 你以前非法抢过票吗，会不会是以前被封禁的

有一定可能性但是我自己手抢没出现这种情况

**Reply by kongfucurry** [reply]
这个被封了12点就不能抢了吗
